src/steps/audio-setup/source-select.tsx

```diff
@@ -38,12 +39,14 @@ export const SourceSelection: React.FC<Props> = ({ selectNoAudio, selectMicropho
         label={t("sources-audio-microphone")}
         onClick={selectMicrophone}
         shortcut={showShortcuts ? SHORTCUTS.audioSetup.withAudio : undefined}
+        isHighContrast={isHighContrast}
       />
       <SourceOptionButton
         icon={<FiMicOff />}
         label={t("sources-audio-without-audio")}
         onClick={selectNoAudio}
         shortcut={showShortcuts ? SHORTCUTS.audioSetup.withoutAudio : undefined}
+        isHighContrast={isHighContrast}
```

Comment:
Why do we pass this as a prop here? The `SourceOptionButton` should probably just use `useColorScheme` itself.